db/migrate/20160329184508_create_gem_download_count_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateGemDownloadCountIndex < ActiveRecord::Migration
+  def change
+    add_index :gem_downloads, [:version_id, :rubygem_id, :count]
```

Comment:
hmm. wonder why I was thinking that `version_id: 0` was in the code. Can't find it so ignore the `version_id` being `0` comment.